nero_view.py

```diff
@@ -45,22 +52,29 @@ def __init__(self) -> None:
         self.modes = []
         self.mode_index = 0
 
-        # for mode_class in (OffMode, PitLaneMode, DebugMode, SpeedMode, EfficiencyMode, ReverseMode):
-        for mode_class in (EfficiencyMode, DebugMode):
+        self.header = Header(parent=container)
+        self.header.place(x=0, y=0, relwidth=1, relheight=0.05)
+        for mode_class in (OffMode, PitLaneMode, EfficiencyMode, SpeedMode, DebugMode, ReverseMode, ChargingMode):
+            # for mode_class in (EfficiencyMode, DebugMode, ChargingMode):
             mode = mode_class(parent=container, controller=self, model=self.model)
             self.modes.append(mode)
-            mode.grid(row=0, column=0, sticky="nsew")
+            mode.grid(row=0, column=0, sticky="s")
 
         self.update_mode()
         self.check_can()
         self.update_buttons()
         self.update_current_page()
         self.last_pinned_update_time = time.time()
         self.update_pinned_data()
+        self.update_header()
 
     def update_mode(self):
-        self.current_mode = self.modes[self.mode_index]
+        if self.model.get_BMS_state() is not None and self.model.get_BMS_state() >= 2:
```

Comment:
why >= 2 what does that even mean

Reply:
this is a decoding thing it means if its plugged in or faulted